18F/identity-idp
# LG-14175: Expire EIPP enrollments in job

## 🎫 Ticket
[LG-14175: Expire EIPP Enrollments in the Job](https://cm-jira.usa.gov/browse/LG-14175)

Related to the first AC in the ticket, here is [the slack conversation](https://gsa-tts.slack.com/archives/C03FA4VBN76/p1723658755557599) where David and I discussed that the error message for expired ID-IPP enrollments and for expired EIPP enrollments have the same message formats.


## 🛠 Summary of changes
- This didn't require any actual changes to the GetUspsProofingResultsJob, because the check that expires ID-IPP enrollments also expires EIPP enrollments.
- I added specs to ensure that EIPP enrollments are expired.
- I made some of the naming for ID-IPP fixtures and specs more precise.



## 📜 Testing Plan
- Come in from Sinatra. Pick Enhanced In-Person Proofing.
- Create an Enhanced IPP enrollment
- Once on the Verify view (/verify/in_person/ready_to_verify), open a new terminal and run: rails c
 - In rails console, grab the enrollment: `e = InPersonEnrollment.last`
Run the below in rails console 
```
e.days_to_due_date
=> 6
> e.update!(enrollment_established_at: e.created_at - 8.days)
e.reload
> e.days_to_due_date
=> -1
```
- Wait for the GetUspsProofingResultsJob to run on the half hour
- Confirm that the user would receive an email that says "The deadline to verify your identity in person has passed". (I included a screenshot of the email below.)
<details>
<summary>Email:</summary>

![Screenshot 2024-08-15 at 16 00 25](https://github.com/user-attachments/assets/bfd52811-4ed5-4b1d-85af-a937ee04a183)
</details>

- Next, reload the enrollment, then check the enrollment status and confirm it is expired
```
e.reload
e.status 
==> “expired”
```
